CHANGELOG.rst

```diff
@@ -1,6 +1,38 @@
 Changelog
 =========
 
+0.8.1 (2018-12-20)
+------------------
+
+MLflow 0.8.1 introduces several improvements:
+
+- Better UI performance, especially when navigating within experiments containing many runs, parameters, and metrics
+- Improved visualizations, including interactive scatter plots for MLflow run comparisons
+- Expanded support for scoring Python models as Spark UDFs
```

Comment:
Do we have docs we can point to for this?

Reply:
Yes, added a link to the feature documentation on mlflow.org: https://mlflow.org/docs/latest/models.html#export-a-python-function-model-as-an-apache-spark-udf